### ISSUE TITLE
predict.py:25-28: Resolve issue with all-time negative...

### ISSUE THREAD

**Issue by 0pdd** [h1alexbel/samples-filter#45]
The puzzle `30-025c8878` from #30 has to be resolved: 

https://github.com/h1alexbel/samples-filter/blob/7d5d392a821b11cf8a7d4ac4216bbd9d91027180/model/predict.py#L25-L28

The puzzle was created by @rultor on 23-Apr-24. 

Estimate: 90 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by h1alexbel** [comment]
optimizing learning rate: `learning_rate=2e-5,` and
reduced batched sizes: `per_device_train_batch_size=8` and `per_device_eval_batch_size=32` showcased this result:

```text
Raw logits: tensor([[ 3.3061, -3.8666]], grad_fn=<AddmmBackward0>)
Probabilities: tensor([[9.9923e-01, 7.6663e-04]], grad_fn=<SoftmaxBackward0>)
Predicted label: negative
```

we should check the result of the model inference on a list of examples.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#52](https://github.com/h1alexbel/samples-filter/issues/52) is still not solved.
